core/services/workflows/store/store_db.go

```diff
@@ -407,5 +463,17 @@ func (d *DBStore) GetUnfinished(ctx context.Context, workflowID string, offset,
 }
 
 func NewDBStore(ds sqlutil.DataSource, lggr logger.Logger, clock clockwork.Clock) *DBStore {
-	return &DBStore{db: ds, lggr: lggr.Named("WorkflowDBStore"), clock: clock}
+	return &DBStore{db: ds, lggr: lggr.Named("WorkflowDBStore"), clock: clock, chStop: make(chan struct{})}
+}
+
+func (d *DBStore) Ready() error {
+	return nil
+}
```

Comment:
Can the state machine implemention be left active?
```suggestion
```

Reply:
done, thanks, I missed it the first time

Reply:
If you use a `services.Engine` instead of the low level `StateMachine`, then more of this boilerplate is wired up automatically:
https://pkg.go.dev/github.com/smartcontractkit/chainlink-common@v0.3.0/pkg/services#Config.NewServiceEngine